group_vars/all/main.yml

```diff
@@ -8,5 +8,6 @@ deployment_static_resources: "{{ deployment_remote_path }}/deploy/files"
 
 static_proj_root: /proj
 
-ngi_group: ngi-sw
+ngi_sw_group: ngi-sw
```

Comment:
Is there a reason to have these as variables if the variable names are identical to their values?

Reply:
maybe not I guess, although I'm in favor of using variables out of principle..